src/NECompletion/CompletionModel.class.st

```diff
@@ -146,9 +146,9 @@ CompletionModel >> sorter: aSorter [
 
 { #category : #accessing }
 CompletionModel >> title [
-	| type |
+	| kind |
 	[
-	type := node receiver propertyAt: #type ifAbsent: [ ^ '' ].
-	^ type name
+	kind := node receiver propertyAt: #type ifAbsent: [ ^ '' ].
```

Comment:
You could also write it

`^ node receiver propertyAt: #type ifPresent: #name ifAbsent: ''`